ethereum/spec/src/testFixtures/java/tech/pegasys/teku/spec/util/RandomChainBuilderForkChoiceStrategy.java

```diff
@@ -113,6 +113,11 @@ public boolean isOptimistic(final Bytes32 blockRoot) {
     return false;
   }
 
+  @Override
+  public boolean isFullyValidated(Bytes32 blockRoot) {
+    return false;
```

Comment:
I think this should be true since it's always returning `false` for `isOptimistic`
```suggestion
    return true;
```

Reply:
right